documentation/API proposals/IMEI Fraud.md

```diff
@@ -5,10 +5,19 @@ IMEI Fraud API submission
 | API name | IMEI Fraud|
 | API owner | MTN - Waseem Amra |
 | Initial API Contributors | MTN - Waseem Amra | 
-| API summary | This API allows for the API Consumer to understand if any fraud markers have been associated with the device. Fraud markers may be applied if the device is lost, suspended, stolen, blacklisted, etc. |
-| Technical viability | This API uses IMEI info, usually stored by the MNO, to determine whether any fraud markers are associated with the device |
+| API summary | This API enables the API Consumer to determine whether fraud has been reported for the connected device. It will use the MSISDN or/ IMEI to check if the associated device has been reported as lost, suspended, stolen, blacklisted. 
+
+Use case – account takeover, online registration  |
+| Technical viability | This API uses IMEI info, usually stored by the MNO, to determine whether any fraud logs are associated with the device. 
+
+Input: MSISDN & or IMEI number 
+
+Output:  
+
+Status: e.g. Blocked, lost,stolen, blacklisted etc 
+Date_reported:10-01-2023  |

```

Comment:
```suggestion
| API summary | This API enables the API Consumer to determine whether fraud has been reported for the connected device. It will use the MSISDN or/ IMEI to check if the associated device has been reported as lost, suspended, stolen, blacklisted. <br>Use case – account takeover, online registration |
| Technical viability | This API uses IMEI info, usually stored by the MNO, to determine whether any fraud logs are associated with the device. <br><li> Input: MSISDN & or IMEI number. <br><li>Output: Status (e.g. Blocked,lost,stolen, blacklisted etc.) & Date_reported:10-01-2023 |
```